perfmetrics/scripts/load_tests/python/tasks/python_os.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains task which reads data from files using python's native open api.
+
+Example:
+
+  lg_obj = load_generator.LoadGenerator(...)
+  task_obj = python_os.OSRead256KB(...)
+  observations = lg_obj.generate_load(task_obj)
+"""
+import os
+import logging
+from load_generator import task
+
+PYTHON_OS_READ = 'python_os_read'
+
+
+class OSRead(task.LoadTestTask):
+  """Task class for reading file from disk using python's native open api.
+
+  The task reads file with os.O_DIRECT i.e. bypassing page cache.
+
+  Args:
+    task_name: String name assigned to task.
+    file_path_format: String format of file path. The format can contain
+     'process_id' and 'thread_id' to be filled. E.g. gcs/256kb/read.{process_id}
+    file_size: Integer size of file in bytes to be read.
+    block_size: Integer size of block in bytes. File is read block by block.
```

Comment:
Please also mention, if we don't pass block_size it will treat file_size as block_size.

Reply:
Done